bolt-modules/boltlib/lib/puppet/functions/catch_errors.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+# Catches errors in a given block and returns them. This will return the
+# output of the block if no errors are raised. Accepts an optional list of
+# error kinds to catch.
```

Comment:
This needs a note calling out it is not available in apply block. See https://github.com/puppetlabs/bolt/blob/ea5b61a8daf363d679ed21225564dfb4fab6ad79/bolt-modules/boltlib/lib/puppet/functions/without_default_logging.rb#L9

Reply:
Fixed